docker/docker-compose.yml

```diff
@@ -0,0 +1,50 @@
+version: "3.5"
+networks:
+  default:
+    name: supernets2
+services:
+  supernets2-mock-l1-network:
+    container_name: supernets2-mock-l1-network
+    image: snapchain/geth-cdk-validium-contracts
```

Comment:
there are now two Dockerfiles. does this image map to the `docker/Dockerfile.geth` one? if yes, there is already a CMD in the dockerfile but we override it here. so it seems that dockerfile is not needed any more?

Reply:
actually should we just use the image at https://hub.docker.com/r/hermeznetwork/geth-cdk-validium-contracts?

Reply:
actually, building our image for the `deploy_output.json`